crates/bevy_sprite/src/render/mod.rs

```diff
@@ -483,7 +484,10 @@ pub struct SpriteViewBindGroup {
     pub value: BindGroup,
 }
 
-#[derive(Component, PartialEq, Eq, Clone)]
+#[derive(Resource, Deref, DerefMut, Default)]
+pub struct SpriteBatches(HashMap<(RetainedViewEntity, MainEntity), SpriteBatch>);
```

Comment:
Do we have an optimised hash of a pair of entities yet? If not then perhaps we need one? In the degenerate case with one batch per sprite I suppose this would hurt. But, sprites should use sprite sheets so that should not happen for large numbers of sprites in practice.

Reply:
I think using `EntityHash` should work no matter how many entities you use in the key. I'll include that instead of the default hasher and confirm.

Reply:
This didn't seem to matter in my testing.